spec/index.html

```diff
@@ -1605,27 +1609,36 @@ <h3>The <code>rdf:JSON</code> Datatype</h3>
         literal value (<a data-cite="INFRA#boolean">`true`, `false`</a>, and <a data-cite="INFRA#nulls">null</a>).
 
         <ul>
-          <li>A <a href="RFC8259#section-4">JSON Object</a> is mapped to a <a data-cite="INFRA#ordered-map">map</a>
+          <li>A <a data-cite="RFC8259#section-4">JSON Object</a> is mapped to a <a data-cite="INFRA#ordered-map">map</a>
             by transforming each object member into a <a data-cite="INFRA#map-entry">map entry</a>
             with the <a data-cite="INFRA#map-key">key</a> taken from the member name and
             <a data-cite="INFRA#map-value">value</a> taken by performing this mapping
             to the member value.
             <a data-cite="INFRA#map-entry">Map entries</a> are treated as being unordered.
-            </li>
-          <li>A <a href="RFC8259#section-5">JSON Array</a> is mapped to a <a data-cite="INFRA#list">list</a>
-            by performing this mapping on each array value.</li>
-          <li>A <a href="RFC8259#section-6">JSON Number</a> is mapped to
+          </li>
+          <li>A <a data-cite="RFC8259#section-5">JSON Array</a> is mapped to a <a data-cite="INFRA#list">list</a>
+            such that this <a data-cite="INFRA#list">list</a> contains as many
+            elements as the <a data-cite="RFC8259#section-5">JSON Array</a> and, for
+            every position |i| in the <a data-cite="RFC8259#section-5">array</a>,
+            the element at the |i|-th position in the
+            <a data-cite="INFRA#list">list</a> is the value that results from
+            applying this mapping to the |i|-th element of the
+            <a data-cite="RFC8259#section-5">array</a>.</li>
+          <li>A <a data-cite="RFC8259#section-6">JSON Number</a> is mapped to
             an <a data-cite="XMLSCHEMA11-2#double"><strong>xsd:double</strong></a>
             using a method consistent with <a data-cite="XMLSCHEMA11-2#f-doubleLexmap">doubleLexicalMap</a>.
```

Comment:
I would add "strictly" here and a note saying that the rounding algorithm there MUST be used.  The reason for these additions is to trump the wording at the end of the box saying.  "Note: This specification permits the substitution of any other rounding algorithm which conforms to the requirements of [[IEEE 754-2008]](https://www.w3.org/TR/xmlschema11-2/#ieee754-2008)."

Reply:
```suggestion
            using a method consistent with <a data-cite="XMLSCHEMA11-2#f-doubleLexmap">doubleLexicalMap</a>,
            which MUST strictly conform to the rounding method described in
            <a data-cite="XMLSCHEMA11-2#f-floatPtRound">floatPtRound</a>.
```